language-support/ts/codegen/tests/src/DA/Test/Daml2Ts.hs

```diff
@@ -94,6 +95,8 @@ tests damlTypes yarn damlc daml2ts davl = testGroup "daml2ts tests"
           groverDaml = grover </> "daml"
           groverDar = grover </> ".daml" </> "dist" </> "grover-1.0.dar"
       createDirectoryIfMissing True groverDaml
+      -- Locked to DAML-LF 1.7 since we get different package ids due to
+      -- package metadata in DAML-LF 1.8.
```

Comment:
Wouldn't it make more sense to update the tests to reflect the new package ids?

Reply:
No, this test is verifying the error message you get if you have two packages with different names but the same package id. `daml build` won’t produce such packages anymore with DAML-LF 1.8 since the package name ends up in the metadata  so the package id is actually different. We could manually patch the `MANIFEST.MF` but that doesn’t really solve anything since `daml2ts` should switch to using the package metadata anyway.

Reply:
I see. That makes sense. Thanks for the explanation.